django/conf/global_settings.py

```diff
@@ -310,6 +310,15 @@
 # file system instead of into memory.
 FILE_UPLOAD_MAX_MEMORY_SIZE = 2621440  # i.e. 2.5 MB
 
+# Maximum size in bytes of request data that will be read before raising a
+# SuspiciousOperation. This applies to accessing request.body and also to any
+# data in form data requests, excluding file uploads.
+DATA_UPLOAD_MAX_MEMORY_SIZE = 2621440  # i.e. 2.5 MB
+
+# Maximum number of keys in a POST request that will be allowed before raising
+# a SuspiciousOperation.
+DATA_UPLOAD_MAX_NUMBER_FIELDS = 100
```

Comment:
I have an admin form on one of my sites where $('form').serialize().split('&').length == 419 fields (a category page with 83 edit-inline rows + extra blank ones). It really isn't an issue at all for memory. Can we start it at 1000?